sections/03_categorize.md

```diff
@@ -167,6 +167,35 @@ advances to practical game-changers.
 Additionally, unique barriers exist in this space that may hinder progress in
 this field.
 
+###### Lack of ground-truth labels in data, and high cost in post-study validation by clinician experts.
```

Comment:
Can we call this something like "The challenge of pyrite standards" to contrast with gold standards that many fields have access to?